src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala

```diff
@@ -125,6 +129,6 @@ class Lambda {
     } yield {
       output.write(inputStream.readAllBytes())
     }
-    Await.result(result, 90.seconds)
+    Await.result(result, 600.seconds)
```

Comment:
With the batching, does this await still need to be this long?

Reply:
It's taking just over 5 mins so I guess depends how much spare space we want - could drop it to 7 or 8 mins?